kzz7yh-f53797/cod-ve09v2_kzz7yh-f53797.xml

```diff
@@ -481,9 +484,9 @@
       corp<lb ed="#V" n="13" break="no"/>eue non fuit factum de costa secundum rationem seminalem. 
    </p>
                <p xml:id="kzz7yh-f53797-d1e909">
-                  <lb ed="#V" n="14"/>Respondeo solet assignari dfentua iter rationes primor 
-      <lb ed="#V" n="15"/>diales et causales et naturales et 
-      semina<lb ed="#V" n="16" break="no"/>les. Quia rationes praimordiales sunt rationes creaturarum factiue in 
+                  <lb ed="#V" n="14"/>Respondeo solet assignari dfentua<!--word--> inter rationes 
```

Comment:
make sure to look at the codex again, to learn the abbreviation for "differentia"

```suggestion
                  <lb ed="#V" n="14"/>Respondeo solet assignari differentia inter rationes 
```